contrib/endpoints/src/api_manager/firebase_rules/firebase_request.cc

```diff
@@ -305,6 +310,8 @@ Status FirebaseRequest::SetNextRequest() {
       auto call = *func_call_iter_;
       external_http_request_.url = call.args(0).string_value();
       external_http_request_.method = call.args(1).string_value();
+      external_http_request_.audience =
+          call.args(call.args_size() - 1).string_value();
```

Comment:
It is not optional right now since there is no default behavior implemented right now. There is no default audience being specified at all.